ckcdeb.h

```diff
@@ -1800,11 +1800,19 @@ _PROTOTYP(int ckxfprintf,(FILE *, const char *, ...));
 #define SIGRETURN return(0)
 #endif /* SIGRETURN */
 
-#ifdef CKNTSIG
-/* This does not work, so don't use it. */
-#define signal ckntsignal
-SIGTYP (*ckntsignal(int type, SIGTYP (*)(int)))(int);
-#endif /* CKNTSIG */
+#ifdef CK_ANSIC
+#ifdef OS2
+#ifdef NT
+typedef SIGTYP (*sig_t)(int);
+#else /* !NT */
+typedef SIGTYP (volatile *sig_t)(int);
```

Comment:
Open Watcom didn't like this line. Other declarations elsewhere used the form below which Watcom is fine with and I assume work
```suggestion
typedef SIGTYP (* volatile sig_t)(int);
```

Reply:
Is the use of volatile required? Does it compile on OS2 without warnings if the volatile is removed?

Reply:
I don't know if its required or not, only that it was added some time after C-Kermit 5A(190), the last version for which source is available. The same appears in two other places in K95 - one in ckotio.c, and twice in ckuus4.c ([one here](https://github.com/KermitProject/ckermit/blob/cbe8041bc40c334426a5982254882bd4f13ba1a7/ckuus4.c#L2616)).

Removing volatile doesn't seem to bother Open Watcom - it builds it without warnings. I'll try to give it a quick test tonight to see if there are any obvious issues with the resulting executable.